libs/mapjet-core/src/lib/utils/resource-loader/resource-loader.ts

```diff
@@ -137,10 +137,10 @@ export class ResourceLoader {
     assets.forEach(asset => asset.remove(this.mapJet!.map));
   }
 
-  private removeLayers(layers: LayerSpecification[]): void {
+  private removeLayers(layers: LayerLoader[]): void {
     layers.forEach(layer => {
-      if (this.mapJet!.map.getLayer(layer.id)) {
-        this.mapJet!.map.removeLayer(layer.id);
+      if (this.mapJet!.map.getLayer(layer.specification.id)) {
```

Comment:
po tej zmianie obawiam się trochę tego operatora not null assertion, który oczekuje wartości innej niż null lub undefined. Mergujmy. Wrzucajmy ;)

Reply:
Zmieniło się tylko odwołanie do idika, raczej nie powinno być problemu z tym ;)